### PR TITLE
Add network settings

### PR DESCRIPTION
**Pull request by OKendigelyan**
## Proposed changes

[Task link](https://app.asana.com/0/1207041888417083/1207811126378425/f)

## Types of changes

- [ ] Bugfix
- [x] New feature
- [ ] Refactor
- [ ] Breaking change
- [ ] UI fix

## Screenshots

https://github.com/user-attachments/assets/2818053a-909e-4a04-b119-67d99a91477f

## Checklist

- [x] Tests that prove my fix is effective or that my feature works have been added
- [ ] Documentation has been added (if appropriate)
- [x] Screenshots are added (if any UI changes have been made)
- [ ] All TODOs have a corresponding task created (and the link is attached to it)

### CODE REVIEW

File: apps/web/src/components/Menu/EditNetworkMenu.test.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+describe("<EditNetworkMenu />", () => {
+  describe("edit mode", () => {
+    beforeEach(() => {
+      store.dispatch(networksActions.upsertNetwork(customNetwork));
+    });
+
+    it("doesn't render name field", async () => {
+      await renderInDrawer(<EditNetworkMenu network={MAINNET} />, store);
+
+      await waitFor(() => expect(screen.queryByLabelText("Name")).not.toBeInTheDocument());
```
Comment (serjonya-trili) [inline apps/web/src/components/Menu/EditNetworkMenu.test.tsx:23]: should we really wait for this to happen? 🤔

File: apps/web/src/components/Menu/EditNetworkMenu.test.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  });
+
+  describe("create mode", () => {
+    describe("name field", () => {
+      it("validates uniqueness", async () => {
+        await renderInDrawer(<EditNetworkMenu />, store);
+        fireEvent.change(screen.getByLabelText("Name"), { target: { value: MAINNET.name } });
+        fireEvent.blur(screen.getByLabelText("Name"));
+        await waitFor(() => {
+          expect(screen.getByText("Network with this name already exists")).toBeInTheDocument();
```
Comment (serjonya-trili) [inline apps/web/src/components/Menu/EditNetworkMenu.test.tsx:95]: toBeVisible please, and in other places

File: apps/web/src/styles/theme.ts [hunk 3]
```diff
@@ -201,6 +219,9 @@ const theme = extendTheme({
         md: { height: "42px", minWidth: "42px", padding: "0 12px" },
         lg: { height: "48px", minWidth: "48px", padding: "0 12px" },
       },
+      defaultProps: {
+        size: "lg",
```
Comment (serjonya-trili) [inline apps/web/src/styles/theme.ts:223]: not sure if it's a good thing. so, now we need to check all the places where the button is used and check we didn't break anything (and ideally, remove size='lg' in such places), if you did feel free to leave it as is

Comment (OKendigelyan) [reply apps/web/src/styles/theme.ts:223]: I removed all size="lg" and double checked the existing buttons, they look fine

**Pull request merged**